contributors/design-proposals/service-policy.md

```diff
@@ -0,0 +1,68 @@
+# Service Policy Proposal
+
+## Abstract
+
+ServicePolicy allows cluster administrators to control the creation and validation of a service.
+The intent of ServicePolicy is to manage the services.
+
+## Motivation
+
+As we know, field level control over allowed fields in services does not exist yet. we can not limit
+the ip range for different users. and if we don't want people use the node port directly, we can not
+implement it although the podsecuitypolicy can limit the "hostPort".
+
+## Goals
+
+1.  Associate [service accounts](../design-proposals/service_accounts.md), groups, and users with
+a set of constraints that dictate how a security context is established for a service.
+1.  Provide the ability for users and infrastructure components to use services with elevated privileges
+on behalf of another user or within a namespace where privileges are more restrictive.
+1.  Secure the ability to reference elevated permissions or to change the constraints under which
```

Comment:
Why would you not use pod secure policy for this?  It’s not clear what you intend.